Dockerfile

```diff
@@ -92,6 +92,12 @@ RUN mkdir -p ${HOME}/.config/matplotlib && \
     echo "backend      : Agg" > ${HOME}/.config/matplotlib/matplotlibrc && \
     python -c "import matplotlib.pyplot"
 
+# Instruct joblib to use disk for temporary files. Joblib defaults to
+# /shm when that directory is present. In the Docker container, /shm is
+# present but defaults to 64 MB.
+# https://github.com/joblib/joblib/blob/0.11/joblib/parallel.py#L328L342
+ENV JOBLIB_TEMP_FOLDER=/tmp
```

Comment:
The reason I think it's worthwhile having here is that this image is the execution environment for Python scripts in the Civis Platform, and is also the default image for container scripts. Without instructing joblib to use /tmp, joblib will fail in an unintuitive way. This means that users will get confusing errors if they try to set `n_jobs` to anything other than 1 in most of scikit-learn, for example.

This only affects joblib memmapping, and without this, joblib memmapping will almost always fail.